build/windows/installer/conf/fluent.conf

```diff
@@ -37,6 +38,14 @@
   </record>
 </filter>
 
+<filter oms.container.log.la>
```

Comment:
Since we have multiple filter plugins. is this filter invoked before record_transformer filter plugin?

Reply:
Looks like from the docs the filters are evaluated top-down. I'll move this filter before the record_transformer one.

Reply:
> Looks like from the docs the filters are evaluated top-down. I'll move this filter before the record_transformer one.

> I see.  can you please point me the doc about "top-down" evaluation for the filter plugins?

Reply:
Yes I read this [here](https://docs.fluentd.org/filter#how-to-use) starting at:
> Like the <match> directive for output plugins, <filter> matches against a tag. Once the event is processed by the filter, the event proceeds through the configuration top-down. Hence, if there are multiple filters for the same tag, they are applied in descending order. Hence, in the following example:

Reply:
order should not matter here, as long as the field 'stream' which this filter looks for is not renamed/removed by other filters.

Reply:
i think, order helps on the perf side for example, in this case, if the logs filtering happens first and then subsequent filters will have less number of records to process, right? This will be helpful in high volume log scenario.

Reply:
Can you point to where you see this ? All of them are chained to same stream of data (data not passed/duped between filters). So as long as data reaches the stream you have it there. Thats why we don't tail excluded namespace files (rather than getting them here & dropping those).